community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md

```diff
@@ -0,0 +1,182 @@
+      - PGADMIN_DEFAULT_PASSWORD=secret
+      - PGADMIN_LISTEN_PORT=8080
+    ports:
+      - "8080:8080"
+    volumes:
+      - pgadmin_volume:/var/lib/pgadmin
+    networks:
+      - backend_network
+volumes:
+  db_volume:
+  pgadmin_volume:
+networks:
+  backend_network:
+    driver: bridge
+```
+
+Save the docker-compose.yaml file.
+
+In this file are two services defined: One for PostgreSQL and one for pgAdmin. Both services have a fix container-name and both have its own volume. In the environments-definition you can see the credentials to access the services. The postgres servuce will be exposed on port 5432 to the outside. pgadmin will be exposed on port 8080 to the outside. Both services are using the same docker network "backend_network", which is also defined in the networks-definition.
+
+# Step 5 - Start all docker services
+To start all docker services run:
+```
+docker-compose up -d
+```
+With the Parameter "-d" all services will start in detach-mode. That means that all services runs in the background without blocking the command-line.
+
+# Step 6 - Checking status of docker containers
+Check the status of the docker containers with the command: ```docker ps -a```
+The output should look like this:
+
+```
+CONTAINER ID   IMAGE                             COMMAND                  CREATED         STATUS                     PORTS                                                        NAMES
+5b9a82c2d403   postgres:14.0                     "docker-entrypoint.s…"   2 minutes ago   Up 2 minutes               0.0.0.0:5432->5432/tcp, :::5432->5432/tcp                    db
+37c02d5fb68c   dpage/pgadmin4:6.1                "/entrypoint.sh"         2 minutes ago   Up 2 minutes               80/tcp, 443/tcp, 0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   pgadmin
+```
+
+In this case the both services are running, both have the status "Up 2 minutes".
+
+# Step 7 - Check logs of the containers
+First check the logs of the postgres database container:
+```
+docker logs db
+```
+
+The log should look like:
+```
+2021-11-07 21:21:06.225 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
+2021-11-07 21:21:06.225 UTC [1] LOG:  listening on IPv6 address "::", port 5432
+2021-11-07 21:21:06.230 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
+2021-11-07 21:21:06.236 UTC [62] LOG:  database system was shut down at 2021-11-07 21:21:06 UTC
+2021-11-07 21:21:06.243 UTC [1] LOG:  database system is ready to accept connections
+```
+
+Then check the logs of the pgadmin container: ```docker logs pgadmin```
+The log should look like:
+```
+[2021-11-07 21:21:35 +0000] [1] [INFO] Starting gunicorn 20.1.0
+[2021-11-07 21:21:35 +0000] [1] [INFO] Listening at: http://[::]:8080 (1)
+[2021-11-07 21:21:35 +0000] [1] [INFO] Using worker: gthread
+[2021-11-07 21:21:35 +0000] [88] [INFO] Booting worker with pid: 88
+```
+
+# Step 8 - Open pgadmin in your browser
+Open the url ```http://localhost:8080/``` in the browser.
```

Comment:
Please change localhost to "your-ip" or something like that

Reply:
done, i pushed the changes